kong/clustering/services/sync/validate.lua

```diff
@@ -8,10 +8,26 @@ local validate_references_sync = declarative_config.validate_references_sync
 local pretty_print_error = declarative.pretty_print_error
 
 
+-- It refers to format_error() function in kong/clustering/config_helper.lua.
+local function format_error(err_t)
+  -- Declarative config parse errors will include all the input entities in
+  -- the error table. Strip these out to keep the error payload size small.
+  local errors = err_t.flattened_errors
+  if type(errors) == "table" then
```

Comment:
if type(errors) ~= "table" then
  return
end

Reply:
fixed, but i didnt fix original code style in kong/clustering/config_helper.lua